doc/src/manual/control-flow.md

```diff
@@ -327,15 +328,20 @@ julia> fact(0)
 
 julia> fact(-1)
 ERROR: n must be non-negative
- in fact(::Int64) at ./none:2
- ...
+[...]
 ```
 
 Boolean operations *without* short-circuit evaluation can be done with the bitwise boolean operators
 introduced in [Mathematical Operations and Elementary Functions](@ref): `&` and `|`. These are
 normal functions, which happen to support infix operator syntax, but always evaluate their arguments:
 
-```julia
+```@meta
```

Comment:
Same remark, again, as in https://github.com/JuliaLang/julia/pull/20183/files#r97537182 can apply here as well I think.